packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -595,4 +596,28 @@ export class Interpreter {
 		}
 		return result;
 	}
+
+	/**
+	 * This functions checks if the requested round is avaiable to access. If it is
+	 * not throws an Error
+	 * @param round: round number
+	 * @returns void
+	 */
+	assertRoundIsAvailable(round: number): void {
+		let firstAvail = this.runtime.ctx.tx.lv - MaxTxnLife - 1;
+		if (firstAvail > this.runtime.ctx.tx.lv || firstAvail === 0) {
```

Comment:
```suggestion
		if (firstAvail > this.runtime.ctx.tx.lv || !firstAvail) {
```

Reply:
I think this one is easier to read because we want explicitly check for zero in this case